client-vms/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@nillion/client-vms",
   "license": "MIT",
   "author": "devsupport@nillion.com",
-  "version": "0.3.0-rc.0",
+  "version": "0.3.1-rc.0",
```

Comment:
it might be nice to break the version bump into its own commit. Totally optional though.